### PR TITLE
Additional map commands

### PR DESCRIPTION
**Pull request by utku-caglayan**
This PR documents the changes on the CLC regarding these changes https://github.com/hazelcast/hazelcast-commandline-client/pull/136

It should be merged, **after the changes above are released**.

### CODE REVIEW

**Comment by JakeSCahill** [comment]
I think we should focus any new content on the dedicated docs: https://github.com/hazelcast/hazelcast-commandline-client/pulls

**Review by yuce — APPROVED** [review]

File: docs/modules/clients/pages/clc.adoc [hunk 1]
```diff
@@ -376,6 +474,103 @@ Example:
...
+Parameters:
+
+- `--name -n` (required): Name of the map.
+- `--key -k` (at least one required): Key of the map entry.
+- `--key-type` (optional): Data type of the key.
+
+Example:
+
+```
+hzc map remove --name mapname --key k1 --key k2
```
Comment (emreyigit) [inline docs/modules/clients/pages/clc.adoc:496]:
```suggestion
hzc map remove-many --name mapname --key k1 --key k2
```

Comment (Serdaro) [inline docs/modules/clients/pages/clc.adoc]: hzc map remove-many ....

**Comment by Serdaro** [comment]
Will be taken care of with https://github.com/hazelcast/hazelcast-commandline-client/pull/149/files#diff-f19fe081576545c396ad7b75e70466ed855004711781cf498e91b14b3640ba08

**Comment by Serdaro** [comment]
Closing this PR since the documentation will reside in https://github.com/hazelcast/hazelcast-commandline-client/tree/main/docs and these updates are already in that repo.

**Pull request closed without merging**